src/FlagPopup.elm

```diff
@@ -233,5 +246,5 @@ flagChoice category title description =
         Element.column NoStyle
             [ spacing 12 ]
             [ bold title
-            , paragraph NoStyle [] [ text description ]
+            , paragraph NoStyle [] [ text <| description ]
```

Comment:
nitpick: this is not needed here
tip: I believe elmjutsu remove these ones automatically for you